sycl/source/detail/device_info.hpp

```diff
@@ -240,15 +240,9 @@ template <> struct get_device_info<bool, info::device::atomic64> {
 
     bool result = false;
 
-    RT::PiResult Err = Plugin.call_nocheck<PiApiKind::piDeviceGetInfo>(
+    Plugin.call_nocheck<PiApiKind::piDeviceGetInfo>(
```

Comment:
Sorry, if I wasn't clear. You do have to check the result for PI_INVALID_VALUE and return "false" (as is you are returning junk in "result").

Reply:
Of course, sorry.  I've updated it now.